values-observer-scs.yaml

```diff
@@ -421,89 +421,6 @@ dnation-kubernetes-monitoring:
           - port: "9180"
             interval: 30s
             path: /metrics
-## Uncomment only when SCS Zuul VM has been bootstrap with node and cadvisor exporters
-    - name: zuul
-      description: SCS Zuul VM
-      jobName: zuul-scs
-      host:
-        address: 213.131.230.42
```

Comment:
I see some references to the `213.131.230.42` in zuul-zookeeper-dashboard. Is this dashboard still relevant?

Reply:
The zuul-zookeeper-dashboard is still relevant and could be used. 
Yep, the pre-defined selection of IP address should not be there, but it is not a major issue and could be removed later.

Reply:
Ok, thank you

Reply:
https://github.com/SovereignCloudStack/k8s-observability/pull/76